openupgradelib/openupgrade_160.py

```diff
@@ -45,3 +58,271 @@ def migrate_translations_to_jsonb(env, fields_spec):
                 if initial_translation_tables == "ir_translation":
                     query = query.replace("_ir_translation", "ir_translation")
                 logged_query(env.cr, query)
+
+
+_BADGE_CONTEXTS = (
+    "secondary", "primary" ,"success", "info", "warning", "danger",
+)
+
+_RTL_REPLACEMENT_CONTEXT = (
+    ("left", "start"),
+    ("right", "end"),
+)
+
+_RTL_REPLACEMENT_ELEMENT = (
+    "text", "float", "border", "border-top", "border-bottom", "rounded",
+)
+
+# These replacements are from standard Bootstrap 4 to 5
+_BS5_REPLACEMENTS = (
+    # Grid stuff
```

Comment:
The comment i use here following the same order in https://getbootstrap.com/docs/5.1/migration